dev-docs/bidders/mediago.md

```diff
@@ -3,11 +3,14 @@ layout: bidder
 title: MediaGo
 description: MediaGo Prebid Bidder Adapter
 biddercode: mediago
-media_types: banner
+media_types: banner,native
+prebid_member: false
```

Comment:
Isn't Baidu a member company now?

Reply:
@ChrisHuie We did join the prebid.org Slack group. But I'm not so sure if that means we are members.

Reply:
I checked with my colleague, and we confirmed that we are members. I will update the code accordingly.